src/test/resources/META-INF/mods.toml

```diff
@@ -65,7 +67,7 @@ loaderVersion="[28,)"
     modId="custom_plant_type_test"
 [[mods]]
     modId="chunk_data_event_save_null_world_test"
-[[dependencies.global_loot_test]]
+[[dependencies.level_change_event_on_enchant_test]]
     modId="forge"
```

Comment:
Why?

Reply:
did this trying to get the test mod launching, but I had used the wrong run config

Reply:
Then don't include it in your PR. Only include intended changes.